### DISCUSSION TITLE
Why can I not use '':" in tags, such as tag: id:001

### DISCUSSION THREAD

**Discussion by zhaojing001** [behave/behave#1080] [Q&A] [answered]
...

**Comment by jenisys** [comment]
Normally, you should provide sufficient context what occurred exactly.

I assume that you are using a `Scenario Outline`/ `Scenario Template` with this tag.
The template mechanism currently prevents you from using some special characters (to prevent you from shooting yourself into the feet).

**Reply by jenisys** [reply]
I just verified that `Scenario Outline`(s) with a `@id:100` works without any problems (using current tip of the repository and behave v1.2.6).

SEE ALSO:
* [behave/model.py: class Tag.make_name](https://github.com/behave/behave/blob/main/behave/model.py#L2134) called by: [ScenarioOutlineBuilder.make_row_tags](https://github.com/behave/behave/blob/main/behave/model.py#L1298)

Therefore, you should provide a minimal example what is not working for you.
In addition, you should provide some information:
* which behave version are you using
* which Python version(s) you are using
* which operating system are you using (just for completeness)

**Comment by lu-maca** [comment]
Hi @jenisys, 

I'm writing here since my question is pretty related to the original one.

Using tags as `@REQ1.positive_case` and `@REQ1.negative_case`, or something like that, is there any way to launch behave with the argument `--tags="@REQ1.*"`, so that it runs all scenarios tagged with `@REQ1.{something}`?

I remember I've read something similar in the past, but I can't remember where.

**Reply by jenisys** [reply, marked as answer]
You need to use one of the prereleases from the Github repo to be able to use `tag-expressions v2`.

SEE ALSO:
* https://github.com/behave/behave/tags
* [docs: new_and_noteworthy_v1.2.7/#tag-expressions-v2](https://behave.readthedocs.io/en/latest/new_and_noteworthy_v1.2.7/#tag-expressions-v2)
* [features/tags.tag_expression_v2.wildcards.feature](https://github.com/behave/behave/blob/main/features/tags.tag_expression_v2.wildcards.feature)
